README.markdown

```diff
@@ -6841,6 +6841,7 @@ If a number-like argument is specified (including strings that look like numbers
 If a non-number-like string argument is specified, then it is interpreted as a "pattern". The following patterns are supported:
 
 * `'*a'`: reads from the socket until the connection is closed. No end-of-line translation is performed;
+* `'*b'`: the BSD style receive, return anything once socket receives;
```

Comment:
@agentzh I think TCP has no "packet" concept unless you have defined a protocol, so "*p" maybe  not good enough. Consider that the receive behavior here is just returning anything the socket got, I suggest to  use `**` (anything) or `*s` (stream) instead.